SwiftI18n/Classes/Views/BaseViews/UIButton+I18n_loc.swift

```diff
@@ -39,7 +39,7 @@ public extension UIButton {
         loc_keysDictionary[state]
     }
 
-    @IBInspectable var locAccessibilityLabelKey: String? {
+    @IBInspectable override var locAccessibilityLabelKey: String? {
```

Comment:
If this is now part of the UIView, why don we need it here as well?
Is there any difference in the code here and in the UIView?
If not, can we remove it.